drivers/sn32/rgb_matrix_sn32f248b.h

```diff
@@ -5,4 +5,7 @@
 
 void init(void);
 void set_color(int index, uint8_t r, uint8_t g, uint8_t b);
-void set_color_all(uint8_t r, uint8_t g, uint8_t b);
\ No newline at end of file
+void set_color_all(uint8_t r, uint8_t g, uint8_t b);
+void shared_matrix_enable(void);
+void shared_matrix_disable(void);
+void rgb_callback(PWMDriver *pwmp);
```

Comment:
Why are the following function exported?
void shared_matrix_enable(void);
void shared_matrix_disable(void);
void rgb_callback(PWMDriver *pwmp);